### ISSUE TITLE
css.properties.list-style-type - <Incorrect counter-style support results>

### ISSUE THREAD

**Issue by r12a** [mdn/browser-compat-data#20222]
### What type of issue is this?

Incorrect support data (example: Chrome says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

I tried several of the built in styles that the page says are supported by Chrome and Edge, and found that they are not, eg. amharic-abegede, and somali.

Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

### What did you expect to see?

Please check all the results.

### Did you test this? If so, how?

https://w3c.github.io/i18n-tests/exploratory/vanilla/index.html?text=%3Cstyle%3Eli%20%7B%20list-style-type%3A%20somali%3B%20margin-inline%3A3rem%3B%20%7D%20%3C%2Fstyle%3E%0A%3Col%3E%0A%3Cli%3Eone%3C%2Fli%3E%0A%3Cli%3Etwo%3C%2Fli%3E%0A%3Cli%3Ethree%3C%2Fli%3E%0A%3Cli%3Efour%3C%2Fli%3E%0A%3Cli%3Efive%3C%2Fli%3E%0A%3Cli%3Esix%3C%2Fli%3E%0A%3Cli%3Eseven%3C%2Fli%3E%0A%3Cli%3Eeight%3C%2Fli%3E%0A%3C%2Fol%3E&fontSize=36&width=500&height=500&a=&i=

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `css.properties.list-style-type`
* Report started: 2023-06-25T05:29:37.349Z

</details>

**Comment by caugner** [comment]
> I tried several of the built in styles that the page says are supported by Chrome and Edge, and found that they are not, eg. amharic-abegede, and somali.

Good catch, it looks like `CSS.supports('list-style-type', 'foo')` returns true.

IIUC this seems to be aligned [with the spec](https://drafts.csswg.org/css-counter-styles/#extending-css2) (emphasis mine):

> If a [<counter-style-name>](https://drafts.csswg.org/css-counter-styles/#typedef-counter-style-name) is used that does not refer to any existing counter style, it must act identically to the [decimal](https://drafts.csswg.org/css-counter-styles/#decimal) counter style (but **does not [compute](https://drafts.csswg.org/css-cascade-5/#computed-value) to decimal**).

That means we cannot use `CSS.supports()` to determine support for `list-style-type` values.

Instead, we would need to manually review all values, and consider them supported only if they render differently than `decimal`.
